client-local/src/main/java/com/cloudera/livy/client/local/LocalClient.java

```diff
@@ -345,7 +302,7 @@ public void run() {
       }
 
       argv.add("--properties-file");
```

Comment:
This can be done in a separate change, but I think we should change the name of this config var. `properties-file` is not descriptive enough. We should call it something like `--spark-conf-file` or something